c/driver_manager/adbc_driver_manager.cc

```diff
@@ -1716,6 +2176,12 @@ AdbcStatusCode AdbcLoadDriver(const char* driver_name, const char* entrypoint,
   return status;
 }
 
+AdbcStatusCode AdbcLoadDriver(const char* driver_name, const char* entrypoint,
+                              int version, void* raw_driver, struct AdbcError* error) {
+  return AdbcFindLoadDriver(driver_name, entrypoint, version, ADBC_LOAD_FLAG_DEFAULT,
+                            raw_driver, error);
+}
```

Comment:
Maybe this should default to no options enabled?

Reply:
I would argue we should default to all options enabled so it searches everywhere for the manifests and applications that use the driver would have to explicitly disable search locations (opt-out, not opt-in). This way it ensures that the default behavior is that the manifest files work and are easily found.